### PR TITLE
Task/admin master layout

### PR DESCRIPTION
**Pull request by smyaseen**
##Goal and Work:

Created  a master layout

### CODE REVIEW

**Comment by ShahirZain** [comment]
please make sure your base branch should be develop

File: src/Features/Auth/ducks.js [hunk 1]
```diff
@@ -1,29 +1,21 @@
...
-    case LOGIN:
-      return {};
-
     case LOGOUT_SUCCESS:
-      return { isLoggedIn: false, token: '', name: '' };
+      return { isLoggedIn: '', accessToken: '', refreshToken: '', name: '' };
 
     case LOGIN_SUCCESS:
-      return { isLoggedIn: true, token: action.payload.token, name: action.payload.name };
+      return {
```
Comment (SiddiquiMonis401) [inline src/Features/Auth/ducks.js:10]: shouldn't it be like that ??
`return {...state, updatedState}`;

File: src/Features/Auth/epics.js [hunk 2]
```diff
@@ -39,10 +41,13 @@ export const signUpEpic = (action$) =>
         body: payload,
       }).pipe(
         mergeMap((res) => {
-          const {
-            tokens: { access },
-          } = res.response;
-          return of(loginSuccess(access));
+          return of(
```
Comment (SiddiquiMonis401) [inline src/Features/Auth/epics.js:44]: Must use destructuring here

File: src/components/AppBar/AppBar.js [hunk 3]
```diff
@@ -17,13 +17,12 @@ const NavBar = () => {
...
   const history = useHistory();
-
   return (
     <StyledDiv>
       <AppBar color="secondary" position="sticky">
         <Toolbar>
           <Grid alignItems="flex-end" container justifyContent="flex-end">
-            <AppBarMenuButton buttonIcon={<MoreVert />}>
+            <AppBarMenuButton>
               <StyledMenuItem onClick={() => history.push('/profile')} theme={theme}>
```
Comment (SiddiquiMonis401) [inline src/components/AppBar/AppBar.js:26]: We should use routename here !!

**Review by SiddiquiMonis401 — CHANGES_REQUESTED** [review]

**Review by SiddiquiMonis401 — APPROVED** [review]

**Pull request merged**